package.json

```diff
@@ -30,9 +30,10 @@
     "lint": "prettier --check ."
   },
   "files": [
-    "bin",
-    "lib",
-    "templates"
+    "bin/fantasticon",
```

Comment:
This change is done automatically by npm because there are 2 `files` properties in master.